README.md

```diff
@@ -38,6 +38,20 @@ SUSI.AI is written in JavaScript with React. To get started with the code, follo
 * node --version >= 6
 * yarn --version >= 3
 
+## Local Installation
+### Steps
+* `git clone <repository-url>` where repository-url is the link to the forked repository
```

Comment:
`git clone <repository-url>`, where `<repository-url>` is the link to the forked repository

Reply:
@akshatnitd Done.